src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -492,6 +493,168 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
         self.name = self.m_def.name if self.name is None else self.name
 
 
+class ParticleCell(Cell):
+    """
+    A base section used to specify the atomic cell information of a system.
+    """
+
+    particles_state = SubSection(sub_section=ParticlesState.m_def, repeats=True)
+
+    n_particles = Quantity(
+        type=np.int32,
+        description="""
+        Number of atoms in the atomic cell.
+        """,
+    )
+
+    equivalent_particles = Quantity(
+        type=np.int32,
+        shape=['n_atoms'],
+        description="""
+        List of equivalent atoms as defined in `atoms`. If no equivalent atoms are found,
+        then the list is simply the index of each element, e.g.:
+            - [0, 1, 2, 3] all four atoms are non-equivalent.
+            - [0, 0, 0, 3] three equivalent atoms and one non-equivalent.
+        """,
+    )
+
+    def __init__(self, m_def: 'Section' = None, m_context: 'Context' = None, **kwargs):
+        super().__init__(m_def, m_context, **kwargs)
+        # Set the name of the section
+        self.name = self.m_def.name  #! self.name here is 'ParticleCell'
+
+    def is_equal_cell(self, other) -> bool:
+        """
+        Check if the atomic cell is equal to an`other` atomic cell by comparing the `positions` and
+        the `AtomsState[*].chemical_symbol`.
+        Args:
+            other: The other atomic cell to compare with.
+        Returns:
+            bool: True if the atomic cells are equal, False otherwise.
+        """
+        if not isinstance(other, ParticleCell):
+            return False
+
+        # Compare positions using the parent sections's `__eq__` method
+        if not super().is_equal_cell(other=other):
+            return False
+
+        # Check that the `chemical_symbol` of the atoms in `cell_1` match with the ones in `cell_2`
+        check_positions = self._check_positions(
+            positions_1=self.positions, positions_2=other.positions
+        )
+        try:
+            for particle in check_positions:
+                type_1 = self.particles_state[particle[0]].particle_type
+                type_2 = other.particles_state[particle[1]].particle_type
+                if type_1 != type_2:
+                    return False
+        except Exception:
+            return False
+        return True
+
+    def get_particle_types(self, logger: 'BoundLogger') -> list[str]:
+        """
+        Get the chemical symbols of the atoms in the atomic cell. These are defined on `atoms_state[*].chemical_symbol`.
```

Comment:
maybe make a TODO that you need to go through and adjust all the atom->particle wording